src/specfem3D/compute_forces_viscoelastic.F90

```diff
@@ -312,6 +317,15 @@ subroutine compute_forces_viscoelastic(iphase,deltat, &
           enddo
         enddo
       enddo
+      !! solving wavefield discontinuity problem with non-split-node scheme
+      !! add back displacement discontinuity for inner elements of
+      !! the discontinuity interface
+      !! note that this is called in adjoint simulation
+      if (IS_WAVEFIELD_DISCONTINUITY .and. &
```

Comment:
this breaks the OpenMP version. 

please add the `IS_WAVEFIELD_DISCONTINUITY` variable to the `$OMP SHARED(..)` section to fix it.